packages/server-core/src/media/static-resource/static-resource.class.ts

```diff
@@ -109,6 +87,6 @@ export class StaticResource extends Service<StaticResourceInterface> {
       const storageProvider = getStorageProvider(params?.query?.storageProviderName)
```

Comment:
We are getting `Storage Provider` using `params?.query?.storageProviderName`. When calling remove method, `storageProviderName` is not being sent in `params.query`. Should we remove `params?.query?.storageProviderName` from `getStorageProvider` method?

Reply:
The intention behind passing storageProviderName to getStorageProvider was to support having multiple storage providers simultaneously. You'd pass storageProviderName to specify which one you wanted to call the operation on; if none is passed, then it defaults to one of them. Since getStorageProvider returns the specific storageProvider object, there's no need to pass storageProviderName to calls to its methods.